Vagrantfile

```diff
@@ -0,0 +1,35 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby ts=2 sw=2 tw=2 et :
+
+
+$script = <<SCRIPT
+VAGGA_CONF_FILE='/home/vagrant/.vagga/settings.yaml'
+if ! grep -q "ubuntu-mirror: " $VAGGA_CONF_FILE
```

Comment:
Please state minimum explanation needed as a comment here with links provided.
The reason should be explicitly clear and obvious from the text.